Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -485,8 +367,29 @@ void SetDisc(std::unique_ptr<DiscIO::VolumeDisc> disc,
     s_auto_disc_change_index = 0;
   }
 
+  // Assume that inserting a disc requires having an empty disc before
+  if (had_disc != has_disc)
+    s_rtc_flag |= RTC_FLAG_DISC_CHANGED;
+
   DVDThread::SetDisc(std::move(disc));
   SetLidOpen();
+
+  s_can_configure_dtk =
+      true;  // TODO: Hardware test needed to confirm that DTK can be configured again
```

Comment:
This looks terrible, would you mind putting the comment in the line _before_ the statement?

Reply:
... jeeze, I didn't see that.  I typed it on one line, and then autoformatted and didn't notice it changed to that monstrosity.